ui/helpers/constants/design-system.ts

```diff
@@ -37,7 +37,6 @@ export enum Color {
   successInverse = 'success-inverse',
   successDisabled = 'success-disabled',
   infoDefault = 'info-default',
-  infoAlternative = 'info-alternative',
```

Comment:
No instances of infoAlternative remain in the codebase

![Screenshot 2024-06-06 at 2 34 41 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/e31781aa-fa59-4e2e-9df4-db5b99b391d6)